src/views/Dashboard/GeneticAnalyst/Account/index.vue

```diff
@@ -857,6 +857,9 @@ export default {
       if (!event.target.value) return
       const file = event.target.files[0]
       
+      if (file.type != "image/jpg" && file.type != "image/png") return this.errorProfile = errorMessages.FILE_FORMAT("PNG/JPG")
```

Comment:
It would be better to make a `constant` array filled with the allowed image types and validate using `includes` rather than typing it out one by one.

Reply:
updated in this commit [bfd2701](https://github.com/debionetwork/debio-genetic-analyst-ui/pull/20/commits/bfd27014c83d684ae2dc0f7787c894e428b35878)